### ISSUE TITLE
Reinstate `ariaRelevant` IDL as custom TokenList reflection

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#1267]
The prose of `aria-relevant` includes some special rules that cannot be expressed in standard WebIDL, so I'm removing its DOMString-based reflection in ARIA 1.2. 

- The default missing value and invalid missing value is actually two values: `additions` and `text`.
- One of the defined values `all` is a synonym of three other values: : `additions`, `removals` and `text`.

It's neither an _enumerated DOMString_, nor a _DOMTokenList_.

See @domenic's comment #1261 about precedent for how to reflect this attribute.
https://github.com/w3c/aria/pull/1261#issuecomment-625502989

I think we should be able to bring this back in 1.3.

**Comment by domenic** [comment]
According to the web platform tests, every browser implements `ariaRelevant` IDL attributes in both `ElementInternals` and `HTMLElement`:

- https://wpt.fyi/results/custom-elements/ElementInternals-accessibility.html?label=master&label=experimental&aligned&q=ElementInternals-accessibility
- https://wpt.fyi/results/html/dom/aria-attribute-reflection.html?label=master&label=experimental&aligned&q=aria-attribute-reflection

Given this, probably we should give up on anything more suited to the data model, and just add it back to the spec, as a simple string attribute?

**Comment by cookiecrook** [comment]
Overcome by https://github.com/w3c/aria/pull/2326
